zeebe/clients/java/src/main/java/io/camunda/zeebe/client/impl/command/UpdateUserTaskCommandImpl.java

```diff
@@ -74,77 +74,73 @@ public UpdateUserTaskCommandStep1 action(final String action) {
   @Override
   public UpdateUserTaskCommandStep1 dueDate(final String dueDate) {
     ArgumentUtil.ensureNotNull("dueDate", dueDate);
-    getChangesetEnsureInitialized().put(Changeset.JSON_PROPERTY_DUE_DATE, dueDate);
+    getChangesetEnsureInitialized().dueDate(dueDate);
```

Comment:
This won't work, unfortunately. This is also why the QA tests are failing. The changeset is handled as a Java Map when serialized. Thus, only the entries in the map itself are serialized. Additional attributes like the `dueDate` are not. See https://github.com/camunda/zeebe/pull/16676#issuecomment-1976393413.

Reply:
Tests are passing now though (locally anyway)

Reply:
AFAIK the property is empty, but it's non null, so we still serialize something in the end.

Reply:
This a POJO class in the client now due to the changes and not a subtype of `HashMap` anymore, right? Than those changes should be fine, yeah, you're right 👍